frontend/src/containers/Homepage/Homepage.js

```diff
@@ -5,7 +5,9 @@ import Footer from '../../components/Footer/Footer';
 import MainSearch from '../../components/MainSearch/MainSearch';
 import RecipeSearchList from '../../components/RecipeSearchList/RecipeSearchList';
 import BottomSignUp from '../../components/BottomSignUp/BottomSignUp';
+import SearchLozenge from '../../components/SearchLozenge/SearchLozenge';
 import './Homepage.css';
+import SearchResults from '../../components/SearchResults/SearchResults';
```

Comment:
I would put the CSS import at the bottom so the component imports are grouped together.